src/components/utils/include/utils/file_system.h

```diff
@@ -241,11 +241,22 @@ const std::string ConvertPathForURL(const std::string& path);
 bool CreateFile(const std::string& path);
 
 /**
- * @brief Get modification time of file
- * @param path Path to file
- * @return Modification time in nanoseconds
+ * @brief Get information of file's last modification time
+ * @param path contains full path to file (including file name and extension)
+ * @return time stamp with file modification time information, which represents
+ * modification time in seconds and nanoseconds
  */
-uint64_t GetFileModificationTime(const std::string& path);
+struct timespec GetFileModificationTime(const std::string& path);
+
+/**
+ * @brief Compare time stamps for file A and file B
+ * @param file_a contains name and timestamp for file A
+ * @param file_b contains name and timestamp for file B
+ * @return true if file A is older than file B, false otherwise
+ */
+bool CompareFilesByModificationTime(
+    std::pair<std::string, struct timespec> file_a,
```

Comment:
@dcherniev I would rather create the meaningful structure like `FileDescriptor` instead of `pair`
```c++
struct FileDescriptor {
std::string name;
timespec modification_time;
}
```